src/objects/Mesh.js

```diff
@@ -245,11 +245,9 @@ Mesh.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 			// Check boundingBox before continuing
 
-			if ( geometry.boundingBox !== null ) {
-
-				if ( ray.intersectsBox( geometry.boundingBox ) === false ) return;
-
-			}
+			if ( geometry.boundingBox === null ) return;
```

Comment:
You now leave the method if no bounding box is defined. This is clearly wrong. The usage of `boundingBox` in `Mesh.raycast()` is optional. If no bounding box is defined, you still want to perform the raycast.

Reply:
If the method does not return when boundingBox is null, it will have additional calculations.I want to reduce the amount of calculation by judging whether the boundingBox intersects.

Reply:
> If the method does not return when boundingBox is null, it will have additional calculations.

The library should perform an intersection test even if no `boundingBox` is defined.

Reply:
you are right